.github/workflows/ci.yml

```diff
@@ -4,7 +4,11 @@
 
 # Run functional regression checks
 name: ci
-on: [push, pull_request]
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
@JosseVanDelm CI complains that the `[ "main" ]` has spaces so I guess just do `["main"]` haha
It happens to all the files but should be fine after.